Makefile

```diff
@@ -483,6 +484,26 @@ test-mssql-migration: migrations.mssql.test migrations.individual.mssql.test gen
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.mssql.test -test.failfast
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.individual.mssql.test -test.failfast
 
+.PHONY: test-e2e
+test-e2e: test-e2e-pgsql
+
+.PHONY: test-e2e\#%
+test-e2e\#%: test-e2e-pgsql\#%
+	echo "Why do I need this?"
+
+# Can I share the database with integration tests? Is it cleaned up between tests?
+.PHONY: test-e2e-pgsql
+test-e2e-pgsql: build generate-ini-pgsql
+	npx playwright install --with-deps
+	GITEA_ROOT="$(CURDIR)" ./$(EXECUTABLE) web -c integrations/pgsql.ini --quiet & \
+	(wait $i && npx playwright test && killall gitea)
```

Comment:
Would suggest a separate bash script like `tools/watch.sh` to launch multiple processes at the same time, that way you also don't need `killall`.